pkgs/development/libraries/libmbim/default.nix

```diff
@@ -11,11 +12,11 @@
 
 stdenv.mkDerivation rec {
   pname = "libmbim";
-  version = "1.24.8";
+  version = "1.26.0";
 
   src = fetchurl {
     url = "https://www.freedesktop.org/software/libmbim/${pname}-${version}.tar.xz";
-    sha256 = "sha256-AlkHNhY//xDlcyGR/MwbmSCWlhbdxZYToAMFKhFqPCU=";
+    sha256 = "1kqkx139z62w391bz6lwmcjg7v12jxlcm7hj88222xrcn8k0j7qy";
   };
 
```

Comment:
Is the `libgudev` buildInput removable now, per the change notes?

Reply:
Right, I missed that part. Removed it, and confirmed it still builds.